src/content/developers/docs/intro-to-ether/index.md

```diff
@@ -13,45 +13,41 @@ To help you better understand this page, we recommend you first read [Introducti
 
-The ether cryptocurrency supports a pricing mechanism for Ethereum's computing power. When users want to make a transaction, they must pay ether to have their transaction recognized on the blockchain. These usage costs are called gas fees, and they depend on the total amount of computing required by a transaction, as well as the network-wide demand for computing when the transaction is submitted.
+Therefore, even if a malicious dapp submitted an infinite loop, the transaction would eventually run out of ether and terminate, allowing the network to return to normal.
 
-Thus, even if a malicious dapp submitted an infinite loop, the transaction would eventually run out of ether and terminate, allowing the network to return to normal.
-
-[More on gas](/developers/docs/gas/)
+It is [common](https://www.reuters.com/article/us-crypto-currencies-lending-insight-idUSKBN25M0GP#:~:text=price%20of%20ethereum) [to](https://abcnews.go.com/Business/bitcoin-slumps-week-low-amid-renewed-worries-chinese/story?id=78399845#:~:text=cryptocurrencies%20including%20ethereum) [conflate](https://www.cnn.com/2021/03/14/tech/nft-art-buying/index.html#:~:text=price%20of%20ethereum) Ethereum and ether — when people reference the "price of Ethereum," they are describing the price of ether.
 
 ## Minting ether {#minting-ether}
 
-Minting is a process in which new ether is created on the Ethereum ledger. The new ether is created by the underlying Ethereum protocol, and it is not possible for a user to create new ether.
+Minting is the process in which new ether gets created on the Ethereum ledger. The underlying Ethereum protocol creates the new ether, and it is not possible for a user to create ether.
 
 Ether is minted when a miner creates a block on the Ethereum blockchain. As an incentive to miners, the protocol grants a reward in each block, incrementing the balance of an address set by the block's miner. The block reward has changed over time, and today it is 2 ETH per block.
 
 ## Burning ether {#burning-ether}
 
-Just as ether can be created by minting, ether can be destroyed by a process called burning. When burn occurs, ether is removed from the Ethereum ledger.
-
-Ether burn occurs in every transaction on Ethereum. When a user pays for their transaction, their base gas fee is automatically destroyed by the Ethereum protocol. [In some blocks](https://etherscan.io/block/12965263), more ether are burned than minted due to base fee burn.
+As well as creating ether through minting, ether can get destroyed by a process called 'burning'. When ether gets burned, it gets removed from circulation permanently.
```

Comment:
Would probably avoid using the term "minting" here given that this term is highly used in the NFT space and could cause confusion... perhaps:
```suggestion
As well as creating ether through block rewards, ether can get destroyed by a process called 'burning'. When ether gets burned, it gets removed from circulation permanently.
```